packages/newspringchurchapp/src/ui/PrayerMenu/__snapshots__/PrayerMenu.tests.js.snap

```diff
@@ -0,0 +1,1034 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`the PrayerMenu component renders the prayer menu 1`] = `
+<RCTSafeAreaView
+  emulateUnlessSupported={true}
+  onLayout={[Function]}
+  style={
+    Object {
+      "bottom": 0,
+      "left": 0,
+      "position": "absolute",
+      "right": 0,
+      "top": 0,
+    }
+  }
+>
+  <View
+    onLayout={[Function]}
+    style={
+      Object {
+        "flex": 1,
+      }
+    }
+  />
+  <View
+    style={
+      Object {
+        "bottom": 0,
+        "left": 0,
+        "position": "absolute",
+        "right": 0,
+        "top": 0,
+      }
+    }
+  >
+    <View
+      style={
+        Object {
+          "paddingHorizontal": 16,
+          "paddingVertical": 16,
+          "zIndex": 2,
+        }
+      }
+    >
+      <Text
+        style={
+          Object {
+            "color": "#303030",
+            "fontFamily": "Colfax-Black",
+            "fontSize": 24,
+            "lineHeight": 27.6,
+          }
+        }
+      >
+        Pray for Others
+      </Text>
+    </View>
+    <View
+      collapsable={false}
+      style={
+        Array [
+          Object {
+            "flex": 1,
+            "overflow": "hidden",
+          },
+          undefined,
```

Comment:
Is this undefined intended?

Reply:
I think this is when the `<TabView>` starts to be rendered. The only place in TAP that I can find this in the the devotional and it does NOT have `undefined` there. We extracted out and are using the React Native `<TabView>`. In TAP, there is a line that wraps the TabView with a `flex: 1` (const withStyles = styled({ flex: 1 }, 'TabView');). I think that's the difference ... so I think this is okay.